Docker/Centos8/Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL maintainer "gluster-users@gluster.org"
 RUN yum -y --setopt tsflags=nodocs --setopt install_weak_deps=0 --enablerepo=powertools install automake autoconf libtool flex bison \
     openssl-devel libxml2-devel libaio-devel readline-devel glib2-devel userspace-rcu-devel \
     libacl-devel fuse-devel redhat-rpm-config rpcgen libtirpc-devel make python3-devel rsync libuuid-devel \
-    rpm-build git attr libcurl-devel selinux-policy-devel firewalld liburing-devel epel-release && yun install -y gperftools-devel
+    rpm-build git attr libcurl-devel selinux-policy-devel firewalld liburing-devel epel-release && yum install -y gperftools-devel
```

Comment:
That's strange - why would you need a separate 'yum install' command for gperftools-devel?

Reply:
epel-release repo needed to be installed first, before we install gperftools, hence the change in command. Couldnt find a better option.